assets/styl/_modules/_navbar.styl

```diff
@@ -24,22 +29,30 @@
         padding 0
 
         a
-            display block
             padding 0 1.8rem
             color $color-base
-            text-align center
-            transition background-color .2s
+            display flex
+
+            *
```

Comment:
`*` ?

Reply:
I used "*" because I want it to match everything but maybe this is too much ^^